### ISSUE TITLE
Add English name to page even if selected is native translate

### ISSUE THREAD

**Issue by bwklol** [NoMansSkyAssistant/General#11]
### English name avaliable
This is a feature request to add English name even if in app is selected native translate. Translates are useful but sometimes they confuses too much and when we really need help with something it would be useful to know the original English name. In example we can look something in native translate that compare with language from game but for further investigate in network would be useful if the app would show the original English name.

In example on this page I've no idea how this product is in English and basically if I need help with it I have to re-do the translation and translate it back to English.

I hope it is understandable since English is not my native. :) 

![image](https://user-images.githubusercontent.com/2501014/100646850-e5b89400-333e-11eb-9728-9cbff9f9a1ac.png)

**Comment by Khaoz-Topsy** [comment]
Yes I think I understand 😋. I wonder would it be useful to be able to set the app language to Polish and set the game items to English? 🤔. Another solution that I can think of, is allowing people to change the app language from this screen, then you can easily translate items between the languages.

**Comment by bwklol** [comment]
> I wonder would it be useful to be able to set the app language to Polish and set the game items to English? 🤔.

That would not be a good solution. Reason why I even use app in Polish is because of polish items names which compare with game language (and I would not want to change whole game language). So it is easy to find them just by looking how they are named in game and the open the app and quickly find them with same name. I just would like to have somewhere below on item page it's original English name. I don't think it would be difficult since the original English name should be available somewhere in the database.

> Another solution that I can think of, is allowing people to change the app language from this screen, then you can easily translate items between the languages.

It could work however I personally think it might be more hard to do rather than just getting out a variable from database, I think
